calicoctl/commands/crds/crds.go

```diff
@@ -0,0 +1,283 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crds
+
+import (
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Same here - I don't think we should be importing the apiextensions-apiserver anywhere in calicoctl.

Reply:
Should be `k8s.io/api`

Reply:
Talked about this offline. The apiextensions CRD objects are not found in the k8s.io/api.